src/syrupy/extensions/amber.py

```diff
@@ -145,17 +158,21 @@ def serialize_dict(
         data: "SerializableData",
         *,
         depth: int = 0,
+        matcher: Optional["PropertyMatcher"] = None,
+        path: "PropertyPath" = (),
         visited: Optional[Set[Any]] = None,
     ) -> str:
-        kwargs = {"depth": depth + 1, "visited": visited}
+        kwargs = {"depth": depth + 1, "matcher": matcher, "visited": visited}
         return (
             cls.with_indent(f"{cls.object_type(data)} {{\n", depth)
             + "".join(
                 f"{serialized_key}: {serialized_value.lstrip(cls._indent)},\n"
                 for serialized_key, serialized_value in (
                     (
                         cls.serialize(**{"data": key, **kwargs}),
```

Comment:
Opted to not include the existing path in the serialization of object keys even though the matcher is included